sources/smallcaps.fea

```diff
@@ -0,0 +1,309 @@
+lookup c2scCapitalstoSmallCapitalslookup1 {
```

Comment:
Make this lookup block a feature block (`feature c2sc {`) and remove the feature blocks below.